scripts/ci/run_targeting_aura_sessions.py

```diff
@@ -0,0 +1,45 @@
+# run `tox -e jupyter-notebook-session-ci`
+
+import logging
+import os
+import random as rd
+import sys
+
+from aura_api_ci import AuraApiCI
+
+logging.basicConfig(level=logging.INFO)
+
+
+def main() -> None:
+    client_id = os.environ["AURA_API_CLIENT_ID"]
+    client_secret = os.environ["AURA_API_CLIENT_SECRET"]
+    tenant_id = os.environ.get("TENANT_ID")
+    aura_api = AuraApiCI(client_id=client_id, client_secret=client_secret, tenant_id=tenant_id)
+
+    MAX_INT = 1000000
+    instance_name = f"ci-build-{sys.argv[1]}" if len(sys.argv) > 1 else "ci-instance-" + str(rd.randint(0, MAX_INT))
+
+    create_result = aura_api.create_instance(instance_name, memory="1GB", type="professional-db")
+    instance_id = create_result["id"]
+    logging.info("Creation of database accepted")
+
+    try:
+        aura_api.check_running(instance_id)
+        logging.info("Database %s up and running", instance_id)
+
+        uri = (create_result["connection_url"],)
+        username = (create_result["username"],)
+        password = (create_result["password"],)
+
+        cmd = f"AURA_DB_ADDRESS={uri} AURA_DB_USER={username} AURA_DB_PW={password} tox -e jupyter-notebook-session-ci"
```

Comment:
Meanwhile, this test runs with an AuraDB instance and dedicated or AuraDS-based sessions.
I suggest renaming it `run_sessions_targeting_auradb`